src/katgpucbf/configure_tools.py

```diff
@@ -0,0 +1,64 @@
+################################################################################
+# Copyright (c) 2021-2025, National Research Foundation (SARAO)
+#
+# Licensed under the BSD 3-Clause License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#   https://opensource.org/licenses/BSD-3-Clause
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+"""Utilities for writing development tools that start CBFs."""
+
+import argparse
+
+
+def parse_develop_options(develop_argument: str) -> dict:
+    """Separate complex develop options into a dictionary."""
+    out_dict = {}
+    for item in develop_argument.split(","):
+        # data_timeout option isn't boolean, unlike the rest
+        if "data_timeout" in item:
+            k, v = item.split("=")
+            out_dict[k] = float(v)
+        else:
+            out_dict[item] = True
+    return out_dict
+
+
+def add_arguments(parser: argparse.ArgumentParser) -> None:
+    """Add common options to an argument parser."""
+    parser.add_argument("--image-tag", help="Docker image tag (for all images)")
+    parser.add_argument("--image-override", action="append", metavar="NAME:IMAGE:TAG", help="Override a single image")
+    parser.add_argument(
+        "--develop",
+        nargs="?",
+        const=True,
+        help="Pass development options in the config. Use comma separation, or omit the arg to enable all.",
+    )
```

Comment:
I see what you were going for with this function name
* Part of me wants to say "Add arguments? Add what arguments?"
* Realistically though, I think the context should be enough (Re: Being in `katgpucbf.configure_tools`)